src/packages/core/components/input-tree/input-tree.element.ts

```diff
@@ -0,0 +1,130 @@
+import { css, html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
+import { FormControlMixin } from '@umbraco-cms/backoffice/external/uui';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbInputDocumentElement } from '@umbraco-cms/backoffice/document';
+
+type NodeType = 'content' | 'member' | 'media';
+export type StartNode = {
+	type?: NodeType;
+	id?: string | null;
+};
+
+@customElement('umb-input-tree')
+export class UmbInputTreeElement extends FormControlMixin(UmbLitElement) {
+	protected getFormElement() {
+		return undefined;
+	}
+
+	private _type: StartNode['type'] = undefined;
+	@property()
+	public set type(newType: StartNode['type']) {
+		if (newType?.toLowerCase() !== this._type) {
+			this._type = newType?.toLowerCase() as StartNode['type'];
+		}
+	}
+	public get type(): StartNode['type'] {
+		return this._type;
+	}
+
+	@property({ type: String })
+	startNodeId?: string;
+
+	@property({ type: Number })
+	min = 0;
+
+	@property({ type: Number })
+	max = 0;
+
+	private _filter: Array<string> = [];
+	@property()
+	public set filter(value: string) {
+		this._filter = value.split(',');
+	}
+	public get filter(): string {
+		return this._filter.join(',');
+	}
+
+	@property({ type: Boolean })
+	showOpenButton?: boolean;
+
+	@property({ type: Boolean })
+	ignoreUserStartNodes?: boolean;
+
+	@property()
+	public set value(newValue: string) {
+		super.value = newValue;
+		if (newValue) {
+			this.selectedIds = newValue.split(',');
+		} else {
+			this.selectedIds = [];
+		}
+	}
+	public get value(): string {
+		return super.value as string;
+	}
+
+	selectedIds: Array<string> = [];
+
+	#onChange(event: CustomEvent) {
+		this.value = (event.target as UmbInputDocumentElement).selectedIds.join(',');
+		this.dispatchEvent(new CustomEvent('change', { bubbles: true, composed: true }));
```

Comment:
The same goes for this event. You can use our own `UmbChangeEvent`. Be aware that our event is not composed, so you might have to check if relied on this event being composed.